bundles/org.openhab.automation.jrubyscripting/src/main/resources/OH-INF/config/config.xml

```diff
@@ -26,14 +26,16 @@
 		<parameter name="gems" type="text" required="false" groupName="gems">
 			<label>Ruby Gems</label>
 			<description><![CDATA[A comma separated list of Ruby Gems to install. Versions may be constrained by separating with an
-			    <tt>=</tt> and then the standard RubyGems version constraint, such as "<tt>openhab-scripting=~>4.0</tt>".
+			    <tt>=</tt> and then the standard RubyGems version constraint, such as "<tt>openhab-scripting=~>5.0.0</tt>".
```

Comment:
Since descriptions go into the translation files, every time we change this, it'll require re-translations. How can we avoid that?

If we do want it here, please regenerate the i18n file with `mvn i18n:generate-default-translations -pl :org.openhab.automation.jrubyscripting`

Reply:
I've updated it to the current iteration, but I don't think we need to update it with every bump of the gem version. it is only a "help text". perhaps only major version bumps?

Reply:
I wasn't suggesting to change it more often. I was thinking of changing that to a generic "gemfile=~>x.x" but having it show openhab-scripting is actually really helpful, so yeah changing it on every major version is OK considering the benefit.